api/internal/localizer/util.go

```diff
@@ -154,3 +156,25 @@ func locRootPath(rootURL string, repoDir string, rootDir filesys.ConfirmedDir) s
 	_, _ = repoDir, rootDir
 	return ""
 }
+
+func loadKustFile(ldr ifc.Loader) ([]byte, string, error) {
```

Comment:
I copied this from `kusttarget`. Let me know if you'd like me to move it to a common package or make it public in `kusttarget`.

Reply:
If this is exactly the same as the `kusttarget` implementation, then IMO you can just make it exportable in `kusttarget`. IIRC it's under `internal` anyways so we wouldn't be exposing it to end-users.

Reply:
Done. I moved the corresponding tests from `localizer_test` to `kusttarget_test`.